cmd/lncli/commands.go

```diff
@@ -463,20 +457,13 @@ func openChannel(ctx *cli.Context) error {
 		return nil
 	}
 
-	if ctx.IsSet("peer_id") && ctx.IsSet("node_key") {
```

Comment:
Maybe return error in case `node_key` is **not** set.

Reply:
This case would be caught by the switch below (it would fall through to "default")